oopfp/lab.py

```diff
@@ -27,17 +27,16 @@ def savings_account(balance):
   basic_account = account(balance)
 
   def dispatch(action, param=""):
-    if(action == 'interest'):
+    if (action == 'interest'):
       return balance * 0.2
     elif (action == 'deposit'):
       return savings_account(balance + 2 * param)
+    elif (action == 'balance'):
+      return balance
```

Comment:
This works, but here we handle the `balance` in our child object, which kind of defeats the purpose of the exercise, right?